docs/develop/parent-to-dialog.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Passing data and messages to a dialog box from its host page
+description: 'Learn how to pass data to a dialog from the host page'
```

Comment:
Could we expand this description to include some more potential SEO terms? Like "Learn how to pass data to a dialog from the host page using the .... APIs ..."?